omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/SessionController1_9Test.java

```diff
@@ -155,4 +161,13 @@ public void post_shouldFailWhenSettingNonexistantLocation() throws Exception {
 		controller.post(hsr, new ObjectMapper().readValue(content, HashMap.class));
 	}
 
+	@Test
+	public void post_shouldSetTheCurrentSession() throws Exception{
```

Comment:
Also, I'd move this so that it is the first of the `post` tests.

```suggestion
	public void post_shouldReturnTheCurrentSession() throws Exception{
```

Reply:
@ibacher I made changes to unit test.